src/Symfony/Bundle/DebugBundle/Command/ServerDumpPlaceholderCommand.php

```diff
@@ -24,15 +25,22 @@
  *
  * @internal
  */
-class ServerDumpPlaceholderCommand extends ServerDumpCommand
+class ServerDumpPlaceholderCommand extends Command
 {
+    private $replacedCommand;
+
     public function __construct(DumpServer $server = null, array $descriptors = array())
```

Comment:
"server" never used. Here for BC? Maybe add a comment about that fact?

Reply:
That's only an internal command, no BC implications here.
No, actually as described in the PR description, this is a placeholder command that matches the  replaced command's constructor signature (`ServerDumpCommand`) [so we can just swap the class](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/DebugBundle/DependencyInjection/DebugExtension.php#L71) when the `ServerDumper` service cannot be wired.